src/constants.js

```diff
@@ -1,3 +1,7 @@
+import English from './content/English'
```

Comment:
Uppercase variables should be reserved for React components only, refactor to `english` and `spanish`